src/rules/style.ts

```diff
@@ -69,6 +79,10 @@ export default {
   'semicolon': [true, 'always', 'ignore-interfaces'],
   // Require or disallow a space before function parenthesis
   'space-before-function-paren': spaceBeforeFunctionParen,
+  // Checks whether the final clause of a switch statement ends in break;.
+  'switch-final-break': true,
+  // Checks that type literal members are separated by semicolons. Enforces a trailing semicolon for multiline type literals.
+  'type-literal-delimiter': false,
```

Comment:
👍 to `false`.  Pretty sure that we favour interfaces over types-with-members, so this rule won't do much for us.

[Examples](https://github.com/palantir/tslint/blob/master/test/rules/type-literal-delimiter/test.ts.lint).